templates/application-qr-code-generator.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: glueops-pull-request-bot
+  finalizers:
+    - resources-finalizer.argocd.argoproj.io
+spec:
+  destination:
+    name: "in-cluster"
+    namespace: glueops-core-pull-request-bot
+  project: glueops-core
+  syncPolicy:
+    syncOptions:
+      - Replace=true
+      - CreateNamespace=true
+    automated:
+      prune: true
+      selfHeal: true
+    retry:
+      backoff:
+        duration: 5s
+        maxDuration: 3m0s
+        factor: 2
+      limit: 2
+    source:
+        repoURL: 'https://helm.gpkg.io/project-template'
+        targetRevision: 0.2.0
+        helm:
+            valueFiles:
+            - values.yaml
+            values: |-
+            replicaCount: '2'
+            image: 
+                registry: ghcr.io
+                repository: glueops/qr-code-generator
+                tag: dev
+                pullPolicy: Always
+                port: 8000
+                
+            deployment: 
+                enabled: true
+
+            service:
+                enabled: true
+                type: ClusterIP
+                ports: 
+                - port: 80
+                    name: service-port
+                    protocol: TCP
+                    targetPort: 8000
+
+            ingress:
+                enabled: true
+                ingressClassName: public
+                entries:
+                - name: external
```

Comment:
I'd probably name this after the app, qr-code-generator

Reply:
resolved

Reply:
It still says `external`. Also, rather than calling it `qr-code-generator` can you call it `public` ? I learned yesterday with Antonio where this name gets used and it gets suffixed to the end of the ingress object name. We have been using `public` in other places so it would just help with consistency

Reply:
@Mbaoma ^